notesapi/v1/tests/test_views.py

```diff
@@ -47,66 +53,38 @@ def setUp(self):
             ],
         }
 
-        self.expected_note = {
-            "created": "2014-11-26T00:00:00+00:00",
-            "updated": "2014-11-26T00:00:00+00:00",
-            "user": TEST_USER,
-            "usage_id": "test-usage-id",
-            "course_id": "test-course-id",
-            "text": "test note text",
-            "quote": "test note quote",
-            "ranges": [
-                {
-                    "start": "/p[1]",
-                    "end": "/p[1]",
-                    "startOffset": 0,
-                    "endOffset": 10,
-                }
-            ],
-            "permissions": {"read": ["group:__consumer__"]},
-        }
-
-    def tearDown(self):
-        Annotation.drop_all()
-
-    def _create_annotation(self, refresh=True, **kwargs):
+    def _create_annotation(self, **kwargs):
         """
-        Create annotation directly in elasticsearch.
+        Create annotation
         """
-        opts = {
-            'user': TEST_USER,
-        }
+        opts = self.payload.copy()
         opts.update(kwargs)
-        annotation = Annotation(**opts)
-        annotation.save(refresh=refresh)
-        return annotation
+        url = reverse('api:v1:annotations')
+        response = self.client.post(url, opts, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        call_command('update_index')
```

Comment:
i'm confused why this is being called every time a note is created in the test - shouldn't the new notes get indexed by the view code automatically?